Std/Lean/Meta/DiscrTree.lean

```diff
@@ -87,6 +87,12 @@ partial def size : Trie α s → Nat
   | Trie.node vs children =>
     children.foldl (init := vs.size) fun n (_, c) => n + size c
 
+/--
+Whether the `Trie` is empty.
+-/
```

Comment:
```suggestion
/-- Whether the `Trie` is empty. -/
-- TODO: remove partial
```

Reply:
It’s as `partial` as the code around it, should I add `TODO`s to the existing functions (`size` for example)?

Reply:
Maybe a TODO at the top of the file will suffice, given how many there are.

Reply:
Added a TODO.

Incidentially, how would I go about unpartializing these functions? They are “morally” structurally recursive, but lean doesn’t see that. So does this need to go via well-founded recursion? If so, do we have to define `Array.attach` (analogous to `List.attach` here, and use it in all these function definitions?

Actually, let me take this to Zulip, I’ll summarize the answer where.

Reply:
In https://github.com/leanprover/std4/pull/332 I am exploring how to make these definitions not `unsafe`; the PRs are independent.